include/mbedtls/mbedtls_config.h

```diff
@@ -3670,6 +3670,9 @@
 //#define MBEDTLS_ECP_WINDOW_SIZE            4 /**< Maximum window size used */
 //#define MBEDTLS_ECP_FIXED_POINT_OPTIM      1 /**< Enable fixed-point speed-up */
 
+/* RSA OPTIONS */
```

Comment:
Minor: this section has the modules in alphabetical order, so please put RSA between PSA and SSL cache.

Reply:
done